src/main/java/com/fundation/webservice/model/CriteriaVideo.java

```diff
@@ -0,0 +1,90 @@
+package com.fundation.webservice.model;
```

Comment:
Please add the provided copyright header.

Reply:
solved